static/scripts/lessons.js

```diff
@@ -0,0 +1,49 @@
+$(document).ready(function() {
+    var $modals = $('.modal');
+    var $deleteModal = $('.delete-modal');
+
+    var reloadLesson = function(href=undefined) {
```

Comment:
function(href=undefined) why? It makes no sense. If an argument is not send it's always undefined. So it's just function(href)